timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -94,31 +100,31 @@ def run(self):
         problem_strings = []
         output_strings = []
 
-        tags_applied[rule.get("file_name")] = 0
+        tags_applied[rule.get("title")] = 0
         try:
             sigma_rule_counter += 1
             tagged_events_counter = self.run_sigma_rule(
                 rule.get("search_query"),
-                rule.get("file_name"),
+                rule.get("title"),
                 tag_list=rule.get("tags"),
-                status_good=rule.get('ts_use_in_analyzer'),
             )
-            tags_applied[rule.get("file_name")] += tagged_events_counter
+            tags_applied[rule.get("title")] += tagged_events_counter
```

Comment:
If I understand the usage of the `def get_kwargs():` function correct, each rule will spawn it's own `run()` instance. If this is true, the `tags_applied` dict would only contain one entry and the value equals `tagged_events_counter`. This is also nowhere used later on, since the output uses `tagged_events_counter` directly. So this dict can probably be removed. wdyt?